### PR TITLE
store label_schema separately from checkpoint

### PR DESCRIPTION
**Pull request by Ilya-Krylov**
depends on https://github.com/openvinotoolkit/training_extensions/pull/751

### CODE REVIEW

File: torchreid/integration/sc/openvino_task.py [hunk 1]
```diff
@@ -197,7 +196,7 @@ def infer(self, dataset: DatasetEntity,
         for i, dataset_item in enumerate(dataset, 1):
             predicted_scene = self.inferencer.predict(dataset_item.numpy)
             dataset_item.append_annotations(predicted_scene.annotations)
-            dataset_item.append_labels(dataset_item.annotation_scene.annotations[0].get_labels())
+            #dataset_item.append_labels(dataset_item.annotation_scene.annotations[0].get_labels())
```
Comment (Ilya-Krylov) [inline torchreid/integration/sc/openvino_task.py]: @sovrasov @kprokofi why did you add this line, looks like it doubles the prediction...

Comment (sovrasov) [reply torchreid/integration/sc/openvino_task.py]: Let's try
```python
            predicted_scene = self.inferencer.predict(dataset_item.numpy)
            dataset_item.append_labels(predicted_scene.annotations[0].get_labels())
```

**Review by sovrasov — APPROVED** [review]
